mlflow/gateway/uc_function_utils.py

```diff
@@ -126,6 +126,22 @@ def is_scalar(function: "FunctionInfo") -> bool:
     return function.data_type != ColumnTypeName.TABLE_TYPE
 
 
+def escape_uc_identifier(full_name: str) -> str:
+    """
+    Escape a Unity Catalog function name for safe SQL interpolation.
+
+    Takes a fully qualified UC name (catalog.schema.function) and escapes each
+    part with backticks to prevent SQL injection.
+    """
+    parts = full_name.split(".")
+    if len(parts) != 3:
+        raise ValueError(
+            f"Invalid Unity Catalog function name format: {full_name}. "
+            "Expected format: catalog.schema.function"
+        )
+    return ".".join(f"`{p.replace('`', '``')}`" for p in parts)
```

Comment:
Why do we use p.replace('`', '``')?

Reply:
To escape backticks, but let me check if a function name can contain them.

Reply:
Ignore this code. I pushed a new commit